laddertools/ranking.py

```diff
@@ -15,11 +15,44 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 
+
 import trueskill
+from math import sqrt, pi, exp, log
+from itertools import count
+from datetime import timedelta
+from collections import deque, defaultdict
+from abc import ABC, abstractmethod
 
 
-class _RatingTrueskill:
+class _RankingBase(ABC):
+    def compute_ratings_from_series_of_games(self, games, player_lookup, **kw):
```

Comment:
Hmm not really, thought it might be useful if we introduce a new ranking system that takes a special input argument that no other ranking system does. If we have the `**kw` present in the signature, the special input argument will be captured by `**kw` and stuff won't crash.

But there's no immediate need for it now, so I might as well remove it to avoid any confusion.